android/shopify_buy_plugin/build.gradle

```diff
@@ -40,6 +40,7 @@ dependencies {
     testCompile 'junit:junit:4.12'
     testCompile 'org.mockito:mockito-core:1.10.19'
     testCompile 'org.json:json:20140107'
+    testCompile 'org.robolectric:robolectric:3.4.2'
```

Comment:
IIRC the purpose of using Robolectric is so you can run unit tests on the JVM while still using Android objects as it provides shims for the Android calls. I've actually gone the other way and made anything referring to Android APIs into an instrument test [1]. The downside is that the tests have to be run from the emulator but when we add in the Unity integration tests we'll probably need to do this anyways. Do you think it makes sense to use an instrument test for this as well?

[1] https://github.com/Shopify/unity-buy-sdk/pull/256